src/Quartz.Tests.Unit/Core/QuartzRandomTest.cs

```diff
@@ -0,0 +1,57 @@
+using NUnit.Framework;
+using Quartz.Core;
+
+namespace Quartz.Tests.Unit.Core
+{
+    [TestFixture]
+    public class QuartzRandomTest
+    {
+        [Test]
+        public void TestNextValidatesPositiveRange()
```

Comment:
I'm not sure if you have any preferences for naming conventions here, but I tried to copy the approach I saw in `QuartzSchedulerTest`. Feel free to let me know if you'd like me to change this